reconcile/cli.py

```diff
@@ -883,11 +883,38 @@ def jenkins_webhooks_cleaner(ctx):
 
 
 @integration.command(short_help="Validate permissions in Jira.")
+@click.option(
+    "--exit-on-permission-errors/--no-exit-on-permission-errors",
+    help="Throw and error in case of board permission errors. Useful for PR checks.",
+    default=False,
+)
+@click.option(
+    "--jira-boards",
+    help="A comma seperated list of Jira board names to operator on. If none is specified, all boards are considered.",
+    required=False,
+)
+@click.option(
+    "--ignored-jira-boards",
+    help="A comma seperated list of Jira board names to be ignored.",
+    required=False,
+)
```

Comment:
these options seem to compete with the approach of the graphql schema, no? i'd expect the integration to decide what to act on based on the data.

Reply:
These options could be used as a last resort to disable the integration for bad-behaving boards. E.g. temporarily disabling the integration due to missing permissions.

Reply:
how about `disable.integration` in a jira board file?

Reply:
Sounds good. Should we honor this attribute in our `jiralert` and `alertmanager` config too? If `jira-permissions-validator` is disabled, no jiralert tickets?

Reply:
i think we have other mechanisms for that, such as labels on app files. but overall, disabling via the data and not via the integration execution is preferred.

Reply:
PTAL https://github.com/app-sre/qontract-schemas/pull/566 and [e0dc02c](https://github.com/app-sre/qontract-reconcile/pull/3957/commits/e0dc02cd3f51ea18fe29143e156b654e24dd7f79)